api/SourceBuffer.json

```diff
@@ -138,6 +138,62 @@
           }
         }
       },
+      "abort_event": {
+        "__compat": {
+          "description": "<code>abort</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SourceBuffer/abort_event",
+          "spec_url": [
+            "https://w3c.github.io/media-source/#dfn-abort",
+            "https://w3c.github.io/media-source/#dom-sourcebuffer-onabort"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "53"
```

Comment:
Chrome 53 is when `onabort` and lots of other event handler properties were added:
https://storage.googleapis.com/chromium-find-releases-static/000.html#00003ed19f12e054784ee59b95c863b380361ef4

I would suggest also including `MediaSource` in this PR, to sort it all out at once.

Unfortunately, we'll have to find the versions when each event was supported.

It also looks like it's the same situation for Edge. Firefox and Safari match the parent feature version though, so those don't need any research.